code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -281,6 +281,13 @@
 				SEND_SIGNAL(src, COMSIG_ADD_MOOD_EVENT, "friendly_hug", /datum/mood_event/betterhug, M)
 		for(var/datum/brain_trauma/trauma in M.get_traumas())
 			trauma.on_hug(M, src)
+	else if(M.zone_selected == BODY_ZONE_PRECISE_MOUTH)
+		playsound(get_turf(src), 'sound/items/toysqueak1.ogg', 50, 1, -1)
+		var/nose_type = isipc(M) ? "screen" : "nose"
```

Comment:
Likely to get cluttered. generalize to a var on species.

Reply:
How do you mean, in the species datum?